code/modules/mob/living/simple_animal/abnormality/waw/judgement_bird.dm

```diff
@@ -103,3 +112,63 @@
 		datum_reference.qliphoth_change(-1)
 	return
 
+/mob/living/simple_animal/hostile/abnormality/judgement_bird/BreachEffect(mob/living/carbon/human/user)
+	..()
+	var/mob/living/simple_animal/hostile/runawaybird/V = new(get_turf(src))
+	birdlist+=V
+	V = new(get_turf(src))
+	birdlist+=V
+	V = new(get_turf(src))
+	birdlist+=V
+
+
+//Runaway birds - Mini Simple Smile, 2 spawned after Jbird kills a player, and 2 on spawn.
```

Comment:
inconsistent comment:
the PR says it spawns 3 when breached and 2 when J-bird claims a life
the comment says it spawns 2 when breached and 2 when J-bird claims a life